package.json

```diff
@@ -9,14 +9,16 @@
     "build": "yarn build-styling",
     "build-styling": "./scripts/build-styling.sh",
     "deploy-storybook": "storybook-to-ghpages",
-    "dist": "lerna run dist",
```

Comment:
Please don't remove this script and relevant `@pika/pack` sections as they are needed.

Reply:
Could you explain why you have choosen picka/pack packages. But anyway, was not my intention to remove them. Just to get this stuff running.

Reply:
These are needed to prepare ES module version and bundled version for CDN.
Check https://github.com/pikapkg/pack for more details.

And yes, the `scss` files are in the separate folders intentionally, so as Pack doesn't attempt to process those (it only picks anything inside `src`).